conf/webhook.routes

```diff
@@ -1,4 +1,4 @@
 # microservice specific routes
 
-POST        /leak-detection             uk.gov.hmrc.platopsapi.leakdetection.LeakDetectionController.webhook()
-POST        /pr-commenter               uk.gov.hmrc.platopsapi.prcommenter.PrCommenterController.webhook()
\ No newline at end of file
+POST        /leak-detection             uk.gov.hmrc.platopsapi.webhook.WebhookController.leakDetection()
+POST        /pr-commenter               uk.gov.hmrc.platopsapi.webhook.WebhookController.prCommenter()
```

Comment:
you could have a single `Action` if you mapped `/leak-detection uk.gov.hmrc.platopsapi.webhook.WebhookController.webhook("leak-detection")`

Reply:
I think I prefer it being explicit - Having a single action means I'd have to handle the string not matching